pontoon/base/models.py

```diff
@@ -94,7 +95,14 @@ def user_display_name(self):
     name = self.first_name or self.email.split('@')[0]
     return u'{name} <{email}>'.format(name=name, email=self.email)
 
+
+@property
+def can_approve_translations(self):
+    return self.groups.filter(permissions__codename='can_translate_locale').exists()
```

Comment:
Just saw the tests below, looks like we can't, huh? The assumption that we assign permissions via groups here is weird, though.

Maybe something like checking if the set returned by `get_objects_for_perm` is empty?